src/containers/stats/demographics/DemographicsGraphs.js

```diff
@@ -98,13 +95,23 @@ const DemographicsGraphs = ({
     else setTimeFrame(option);
   };
 
-  const getMonthlyDemographicsData = () => {
-    actions.getMonthlyDemographics({ month: month.value, year: year.value });
+  const onChangeMonth = (newMonth :Object) => {
+    setMonth(newMonth);
+    actions.getMonthlyDemographics({ month: newMonth.value, year: year.value });
   };
 
+  const onChangeYear = (newYear :Object) => {
+    setYear(newYear);
+    actions.getMonthlyDemographics({ month: month.value, year: newYear.value });
+  };
```

Comment:
same comment here about calling `actions.getMonthlyDemographics` only when the parameter changes